client/src/components/FlagshipMenu/Flagship.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react'
+import './Flagship.css'
+import Icons from '../Icons/FAIcon'
+
+const Menu = (props) => {
+	return(
+		<div className='flagship-container'>
+			<header>
+				<Icons name={['fas', 'arrow-left']} size='xs' />
+				<h2 className='flagship-city'>{props.city}</h2>
+			</header>
+			<img />
+			<section>
+				<h3>Store Contact Info</h3>
+				<hr className='flagship-line'/>
+				<div className='flagship-info'>
+					<Icons name={['fas', 'map-marker-alt']} />
+					<h4 className='contact-info-title'>Address</h4>
+					<br/>
+					<p className='contact-info'>{props.address}</p>
+				</div>
+				<div className='flagship-info'>
+					<Icons name={['fas', 'phone-alt']} />
+					<h4 className='contact-info-title'>Phone</h4>
+					<br/>
+					<p className='contact-info'>{props.phone}</p>
+				</div>
+				<div className='flagship-info'>
+					<Icons name={['far', 'envelope']} />
+					<h4 className='contact-info-title'>Email</h4>
+					<br/>
```

Comment:
Can you achieve this with CSS rather than using the `br` tag. Never use the `br` tag